src/modules/universe/RadixNodeConnection.ts

```diff
@@ -150,12 +149,63 @@ export class RadixNodeConnection extends events.EventEmitter {
         return subscription
     }
 
+    /**
+     * Unsubscribe for all existing and future atoms for a given address
+     * 
+     * @param address - Base58 formatted address
+     * @returns A promise with the result of the unsubscription call
+     */
+    public unsubscribe(address: string): Promise<any> {
+        const subscriberId = this._addressSubscriptions[address]
+
+        return new Promise((resolve, reject) => {
+            this._socket
+                .call('Atoms.cancel', {
+                    subscriberId,
+                })
+                .then((response: any) => {
+                    logger.info(`Unsubscribed for address ${address}`)
+
+                    delete this._addressSubscriptions[address]
+
```

Comment:
You should also call .complete on the corresponding subject as well, so that it can be garbage collected

Reply:
Gotcha